### DISCUSSION TITLE
libiraty tuist errir

### DISCUSSION THREAD

**Discussion by Roy-wonji** [pointfreeco/swift-composable-architecture#2582] [General]
### Description

We received an error that we couldn't handle:
    - Localized description: The data couldn’t be read because it isn’t in the correct format.
    - Error: dataCorrupted(Swift.DecodingError.Context(codingPath: [CodingKeys(stringValue: "targets", intValue: nil), _JSONKey(stringValue: "Index 2", intValue: 2), CodingKeys(stringValue: "type", intValue: nil)], debugDescription: "Cannot initialize TargetType from invalid String value macro", underlyingError: nil))

If you think it's a legit issue, please file an issue including the reproducible steps: https://github.com/tuist/tuist/issues/new/choose
Consider creating an issue using the following link: https://github.com/tuist/tuist/issues/new/choose
make: *** [fetch] Error 1  
tuist fetch libiray error

.remote(url: "http://github.com/pointfreeco/swift-composable-architecture", requirement: .upToNextMajor(from: "1.3.0"))   how to fix

### Checklist

- [x] I have determined whether this bug is also reproducible in a vanilla SwiftUI project.
- [X] If possible, I've reproduced the issue using the `main` branch of this package.
- [X] This issue hasn't been addressed in an [existing GitHub issue](https://github.com/pointfreeco/swift-composable-architecture/issues) or [discussion](https://github.com/pointfreeco/swift-composable-architecture/discussions).

**Comment by mbrandonw** [comment]
Hi @Roy-wonji, this seems to be an issue with Tuist, not our library. Have you tried opening an issue with them? 

Since it's not an issue with this library I am going to convert it to a discussion.
